### PR TITLE
EXE Signing, MSI Creation, MSI Signing

### PR DESCRIPTION
**Pull request by luke-hagar-sp**

### CODE REVIEW

File: build/windows/sail.wxs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        <Package Compressed="yes" InstallerVersion="$(var.InstallerVersion)"
+            InstallScope="perMachine" />
+        <MediaTemplate EmbedCab="yes" />
+
+        <!-- Remove older product(s) early but within the transaction -->
+        <MajorUpgrade Schedule="afterInstallInitialize"
+            DowngradeErrorMessage="A newer version of !(bind.property.ProductName) is already installed." />
+
+        <!-- Upgrade older x86 products -->
+        <Upgrade Id="7C0A5736-5B8E-4176-B350-613FA2D8A1B3">
```
Comment (philip-ellis-sp) [inline build/windows/sail.wxs:38]: Is this ID somehow specific to our installation? Maybe this isn't even needed, not sure

**Review by philip-ellis-sp — APPROVED** [review]

File: build/windows/sail.wxs (hunk 1 above)
Comment (luke-hagar-sp) [reply build/windows/sail.wxs:38]: Oh Jeez, I really have no idea, Maybe thats something we need to generate?

Comment (luke-hagar-sp) [reply build/windows/sail.wxs:38]: https://stackoverflow.com/questions/23529029/wix-generating-new-upgrade-code

Any chance you could help me make sense of all this?

Comment (luke-hagar-sp) [reply build/windows/sail.wxs:38]: Ok so these values are GUIDs for different versions of your app on different platforms like arm, amd64 stuff like that, Do you know how we can generate those GUIDs?

**Pull request merged**